experimental/adaptive-tool/server/qna/src/providers/diagnostics.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+	Diagnostic,	Connection, DiagnosticSeverity
+} from 'vscode-languageserver';
+
+import * as util from '../util';
+import * as path from 'path';
+
+import { TextDocument } from 'vscode-languageserver-textdocument';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const parseFile = require('@microsoft/bf-lu/lib/parser/lufile/luParser');
+
+
+
+export async function updateDiagnostics(document: TextDocument, connection: Connection): Promise<void> {
```

Comment:
Why all the blank lines?

Reply:
Thanks for the review. Optimized the style issues.